frontend/README.md

```
@@ -9,3 +9,19 @@ pnpm create astro@latest -- --template with-vitest
+
+Pour configurer correctement l'application, créez un fichier `.env` à la racine du projet frontend avec les variables suivantes :
+
+```env
+# URL de base du site pour les URLs canoniques
+SITE_URL=https://votresite.com
+
+# Pour le développement local, vous pouvez utiliser :
+# SITE_URL=http://localhost:4321
+```
+
+### Variables disponibles :
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ' :'

(MD026, no-trailing-punctuation)